### ISSUE TITLE
Cannot connect via SSH

### ISSUE THREAD

**Issue by RosaEinhorn** [Ravinou/borgwarehouse#128]
Hi everyone,
thanks a lot for creating this awesome project that I just stumbled across.

I spun it up using the documentation. However, I can't connect to a repo via SSH. I have the feeling that somethings fails during startup. The only few log lines that I get are:
````
docker compose up
[+] Running 2/2
 ✔ Network borg_default     Created                                                                                                                                                    0.1s 
 ✔ Container borgwarehouse  Created                                                                                                                                                    0.1s 
Attaching to borgwarehouse
borgwarehouse  | /etc/ssh is empty, generating SSH host keys...
borgwarehouse  | ssh-keygen: generating new host keys: RSA ECDSA ED25519 
borgwarehouse  | The authorized_keys file does not exist, creating...
borgwarehouse  | Getting SSH fingerprints...
````

The logs are not indicating any issue:
````
cat borgwarehouse.log 
2024-01-05T16:20:33.075285+00:00 BorgWarehouse   ▲ Next.js 13.5.6
2024-01-05T16:20:33.075753+00:00 BorgWarehouse   - Local:        http://7d6f44de2492:3000
2024-01-05T16:20:33.075761+00:00 BorgWarehouse   - Network:      http://172.18.0.2:3000
2024-01-05T16:20:33.075780+00:00 BorgWarehouse  ✓ Ready in 162ms
````
````
cat sshd.log 
2024-01-05T16:20:32.511940+00:00 sshd Server listening on 0.0.0.0 port 22.
2024-01-05T16:20:32.512414+00:00 sshd Server listening on :: port 22.
2024-01-05T16:32:15.930126+00:00 sshd Received signal 15; terminating.
````

Any idea on how to dig deeper to find the issue?

Thanks a lot in an advance. :)

**Comment by Ravinou** [comment]
Hi @RosaEinhorn 😀

When you said _"I can't connect to a repo via SSH"_, what do you mean ? Did you try to connect to your repository with a shell access ?

**Comment by Ravinou** [comment]
@RosaEinhorn tell me if I can close this issue or if you have any questions please 😉

**Comment by RosaEinhorn** [comment]
Hi @Ravinou, thanks for getting back to this and sorry for my late replay.
I cannot replicate the issue anymore. What I tried first was to run borgwarehouse using the prebuilt image which led to the issue described: I was not able to connect to borgwarehouse via ssh, the WebUI worked well.
What I then tried was: I configured the bare metal installation which worked fine.
After that I used the docs to bild the container myself using the correct PID and GID which worked fine as well.
After that I did another try using the prebuilt image which worked as well.
Unfortunately, I could not find out the specific issue. I think it must habe beed some configuration problem from my side. :)
